base/shell/explorer/trayntfy.cpp

```diff
@@ -1309,7 +1309,6 @@ class CTrayNotifyWnd :
     SIZE szTrayClockMin;
     SIZE szTrayNotify;
     MARGINS ContentMargin;
-    HFONT hFontClock;
```

Comment:
Clang is stricter on unused stuff?
lgtm.

Reply:
IIRC it prints warnings on such things.